tests/hikari/test_templates.py

```diff
@@ -37,7 +43,53 @@ def obj(self):
             updated_at=object(),
             source_guild=object(),
             is_unsynced=True,
+            app=mock_app,
```

Comment:
```suggestion
            app=mock.Mock(),
```
No real need for the extra fixture + spec_set. Also, as before, moving to the top

Reply:
Done